### DISCUSSION TITLE
2 Active spools for IDEX printers

### DISCUSSION THREAD

**Discussion by GentlePersuAZN** [Donkie/Spoolman#397] [Ideas]
Would it be possible to add a second active spool for IDEX printers?

**Comment by dohlin** [comment]
This would be awesome!  Just finished up my IDEX installation and just now realizing tracking isn't gonna work so hot :(

**Comment by Donkie** [comment]
Spoolman supports multiple active spools without problem. It's up to your controller to handle it correctly, e.g. Moonraker.

**Reply by dohlin** [reply]
Thank you.  I'll have to do some looking into that as to date I must have not found any doc on setting that up but I'll keep looking.
